### PR TITLE
Add mergeScripts to generate `roll20.sh`

### PR DESCRIPTION
**Pull request by jazeee**
Add a script to merge Roll20 javascript into a single file.

Add a test case to verify that this occurs.

### CODE REVIEW

File: ui/bin/roll20.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
+(function() {
```
Comment (jazeee) [inline ui/bin/roll20.js]: This `bin` file is auto-generated on commit via the pre-commit hook.
We should also be able to do this via github actions: https://docs.github.com/en/actions/quickstart

Comment (jazeee) [reply ui/bin/roll20.js]: https://github.com/epic-power-rpg/roll20/pull/13

**Comment by jazeee** [comment]
Just a ping on this PR. @johnlamping, what are your thoughts on auto-squashing the JS in this PR?

**Review by johnlamping — APPROVED** [review]

**Comment by johnlamping** [comment]
I apologize for dropping the ball on this.

**Comment by jazeee** [comment]
No prob.

**Pull request merged**